pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -383,9 +385,33 @@ func isConfiguredToOwnCloudController(cm *corev1.ConfigMap) (bool, error) {
 	return len(cloudProvider) != 1 || (cloudProvider[0] != "external" && cloudProvider[0] != ""), nil
 }
 
-func manageKubeControllerManagerConfig(ctx context.Context, client corev1client.ConfigMapsGetter, recorder events.Recorder, operatorSpec *operatorv1.StaticPodOperatorSpec) (*corev1.ConfigMap, bool, error) {
+func manageKubeControllerManagerConfig(ctx context.Context, client corev1client.ConfigMapsGetter, infrastuctureLister configv1listers.InfrastructureLister, recorder events.Recorder, operatorSpec *operatorv1.StaticPodOperatorSpec) (*corev1.ConfigMap, bool, error) {
```

Comment:
if this should be implemented, this has to be done through config observer. This place just merges everything and writes the final config map